server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -107,6 +112,85 @@ public int getBalancerComputeThreads()
     return balancerComputeThreads;
   }
 
+  @JsonProperty
+  public Set<String> getKillDataSourceWhitelist()
+  {
+    return killDataSourceWhitelist;
+  }
+
+  @Override
+  public String toString()
+  {
+    return "CoordinatorDynamicConfig{" +
+           "millisToWaitBeforeDeleting=" + millisToWaitBeforeDeleting +
+           ", mergeBytesLimit=" + mergeBytesLimit +
+           ", mergeSegmentsLimit=" + mergeSegmentsLimit +
+           ", maxSegmentsToMove=" + maxSegmentsToMove +
+           ", replicantLifetime=" + replicantLifetime +
+           ", replicationThrottleLimit=" + replicationThrottleLimit +
+           ", balancerComputeThreads=" + balancerComputeThreads +
+           ", emitBalancingStats=" + emitBalancingStats +
+           ", killDataSourceWhitelist=" + killDataSourceWhitelist +
+           '}';
+  }
+
+  @Override
+  public boolean equals(Object o)
```

Comment:
they would have to update the unit test to include the new constructor parameter. If they update the constructor parameter but don't update the test itself then hopefully someone reviewing the PR would notice.